platform/security/servlet/security-servlet-session-expiry/src/main/java/org/codice/ddf/security/servlet/expiry/SessionManagementService.java

```diff
@@ -126,7 +126,7 @@ public Response getRenewal(@Context HttpServletRequest request) {
   public Response getInvalidate(@Context HttpServletRequest request) {
     StringBuffer requestURL = request.getRequestURL();
     String requestQueryString = request.getQueryString();
-    return Response.temporaryRedirect(
+    return Response.seeOther(
```

Comment:
Are we sure that we don't want this to be something like a `permanentRedirect` or `movedPermanently`?

Reply:
According to the HTTP status codes, I think See Other is what we want. I don't think we want to send something like a 301 for this. https://en.wikipedia.org/wiki/HTTP_301

Reply:
I concur. This is not a permanent redirect. It is a forcible redirect under certain conditions (session expiry). This is analogous to what something like a banking website might do.

Reply:
Works for me. Figured I'd bring it up to help make sure that we're not changing it again in a few months.